docs/device-manager.md

```diff
@@ -0,0 +1,40 @@
+PingViewer automatically scans for available devices on serial ports (COM) and UDP connections. If a Ping device is detected, the available connection is shown and the user will be able to choose between all available devices in **Device Manager**.
+
+PingViewer attempts to connect to UDP port 9090 [Ping1D] and 9092 [Ping360] on the host at IP address 192.168.2.2. This is the default IP address of the **Companion computer** that runs on the BlueROV2. The **Companion computer** automatically configures a communication bridge on these ports to a connected sensor.
```

Comment:
"ports" instead of "port"?

Reply:
9090 and 9092, two ports.

Reply:
"to UDP **ports** 9090 [Ping1D] and 9092 [Ping360] "
I would also use parenthesis instead

Reply:
Done.